src/router/index.js

```diff
@@ -24,6 +25,11 @@ export default new Router({
       path: '/live',
       name: 'live',
       component: Live
+    },
+    {
+      path: '/feed',
```

Comment:
I thought that the path on this screen would be the root.
Is not it better?

Reply:
Yes, but i only use it temporarily, after integrating animation we'll not use router.